certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/dto/value/CertificateDataValueDateRange.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
```

Comment:
Remove licence code

Reply:
Since I can search for where it's used you can add this comment once instead of every occasion. It gets hard to see which comments actually touches code and not the license.